aws/table_aws_pricing_product.go

```diff
@@ -0,0 +1,274 @@
+			{Name: "physical_processor", Description: "Describes the processor on your Amazon EC2 instance.", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.physicalProcessor")},
+			{Name: "clock_speed", Description: "Describes the operating speed of your AWS instances.", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.clockSpeed")},
+			{Name: "service_name", Description: "A simplified description about the AWS service.", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.servicename")},
+			{Name: "instance_type", Description: "Describes the instance type, size, and family, which define the CPU, networking, and storage capacity of your instance.", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.instanceType")},
+			{Name: "tenancy", Description: "The type of tenancy allowed on the Amazon EC2 instance.", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.tenancy")},
+			{Name: "license_model", Description: "Describes the license model for your instance.", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.licenseModel")},
+			{Name: "current_generation", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.currentGeneration")},
+			{Name: "pre_installed_software", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.preInstalledSw")},
+			{Name: "processor_architecture", Description: "Describes your processor architecture.", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.processorArchitecture")},
+			// multi az attributes
+			{Name: "deployment_option", Description: "", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.deploymentOption")},
+			// RDS fields
+			{Name: "database_engine", Description: "", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.databaseEngine")},
+			{Name: "min_volume_size", Description: "", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.minVolumeSize")},
+			{Name: "maxVolumeSize", Description: "", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.maxVolumeSize")},
+			// Elasticache fields
+			{Name: "cache_engine", Description: "", Type: proto.ColumnType_STRING, Transform: transform.FromField("Product.Attributes.cacheEngine")},
+			// all attributes
+			{Name: "term_attributes", Type: proto.ColumnType_JSON, Transform: transform.FromField("Product.Attributes")},
```

Comment:
If we're parsing out the attributes above, I'm not sure we should include this column along with them. It feels like duplicate info, and I imagine this field could be pretty large. Is there a particular use case you thought of when including this?

Reply:
You're right : for rds, ec2, and elasticache instances, it's duplicate info.

My goal was to allow users to fetch their custom attributes for a specific service_code which is not already supported by the table.

I think adding others columns is better, even if there will be a lot of nil values